README.md

```diff
@@ -1,86 +1,257 @@
 <p align="center">
   <a href="https://flow-launcher.github.io">
-	<img width="500px" src="https://github.com/Flow-Launcher/Flow.Launcher/blob/5ba4514f31e624c679628d4dfe89036c0e24006c/Doc/Logo/resources/flow-header-square-transparent.png">
+	<img src="https://user-images.githubusercontent.com/6903107/144517044-6c804af0-f331-4ffc-95f7-c3bcb4cc71e0.png" width="650">
   </a>
 </p>
 
 ![Maintenance](https://img.shields.io/maintenance/yes/3000)
+[![Crowdin](https://badges.crowdin.net/flow-launcher/localized.svg)](https://crowdin.com/project/flow-launcher)
 [![Build status](https://ci.appveyor.com/api/projects/status/32r7s2skrgm9ubva?svg=true&retina=true)](https://ci.appveyor.com/project/JohnTheGr8/flow-launcher/branch/dev)
 [![Github All Releases](https://img.shields.io/github/downloads/Flow-Launcher/Flow.Launcher/total.svg)](https://github.com/Flow-Launcher/Flow.Launcher/releases)
 ![GitHub Release Date](https://img.shields.io/github/release-date/Flow-Launcher/Flow.Launcher)
 [![GitHub release (latest by date)](https://img.shields.io/github/v/release/Flow-Launcher/Flow.Launcher)](https://github.com/Flow-Launcher/Flow.Launcher/releases/latest)
 [![Documentation](https://img.shields.io/badge/Documentation-7389D8)](https://flow-launcher.github.io/docs)
 [![Discord](https://img.shields.io/discord/727828229250875472?color=7389D8&labelColor=6A7EC2&label=Community&logo=discord&logoColor=white)](https://discord.gg/AvgAQgh)
 
-Flow Launcher. Dedicated to make your workflow flow more seamlessly. Aimed at being more than an app launcher, it searches, integrates and expands on functionalities. Flow will continue to evolve, designed to be open and built with the community at heart.
 
-<sub>Remember to star it, flow will love you more :)</sub>
+Search everything from applications, files, bookmarks, YouTube, Twitter and more. All from the comfort of your keyboard without ever touching the mouse.
```

Comment:
would this be better?
`Dedicated to make your workflow flow more seamlessly. Search everything from applications, files, bookmarks, YouTube, Twitter and more. Flow will continue to evolve, designed to be open and built with the community at heart.`

Reply:
I think it's too poetic. I'll change it first.

Reply:
> I think it's too poetic. but I'll change it first.

I think at the top, it should simply say what this program does. The phrase you write down feels like a philosophy of development. It is okay to have this phrase written after that. But as I said yesterday, I can't think of a simple phrase for this program.